### DISCUSSION TITLE
What is the avatar of the authors section for?

### DISCUSSION THREAD

**Discussion by eyllanesc** [HEIGE-PCloud/DoIt#364] [Q&A] [answered]
What is the avatar of the authors section for?, by example

https://github.com/HEIGE-PCloud/DoIt/blob/27f1ea3274d060f13b9c6f268ff6d7321f977258/exampleSite/data/authors/PCloud.toml#L4

I thought it would be shown in the authors section but it only shows the following:

https://hugodoit.pages.dev/authors/

![Screenshot_20211127_135630](https://user-images.githubusercontent.com/4896200/143718884-7a3b5357-e5bb-4737-876f-a37da3b664fb.png)

https://hugodoit.pages.dev/authors/pcloud/

![Screenshot_20211127_135802](https://user-images.githubusercontent.com/4896200/143718916-417f3e5b-1b5a-4357-bb25-e0c6729acf6a.png)

**Comment by HEIGE-PCloud** [comment, marked as answer]
It is intended to be used for the sponsor section #275 

Maybe the avatar can be added to the author taxonomy page as well……?

**Reply by eyllanesc** [reply]
@HEIGE-PCloud Okay, I see. But I was hoping it will also be used in the authors section. I hope you can implement it.

**Reply by eyllanesc** [reply]
I think that you should separate the avatar of the sponsor and the avatar of the author. For example, there would be the option not to place the image in the sponsor.
